tests/unit/test_charm.py

```diff
@@ -58,8 +58,67 @@ def test_on_config_changes_aborts_if_host_is_container(harness, mocker):
     assert not harness.charm._stored.started
 
 
+def test_get_initiator_name_from_file(harness, mocker, tmp_path):
+    """Test fetching the initiator name from initiatorname.iscsi file."""
+    initiator_content = dedent(
```

Comment:
**suggestion (non-blocking):** 

I suggest to move this content as global static variable which is defined outside this function.